packages/quick-start-browser/src/BrowseFiles.tsx

```diff
@@ -0,0 +1,51 @@
+import { Icon, Link, Text } from "@fluentui/react";
+import { DocumentIcon, UploadIcon } from "@fluentui/react-icons-mdl2";
+import { registerIcons } from "@fluentui/react/lib/Styling";
+import React from "react";
+import { useDropzone } from "react-dropzone";
+
+registerIcons({
+    icons: {
+        upload: <UploadIcon />,
+        doc: <DocumentIcon />,
+    },
+});
+
+interface BrowseFilesProps {
+    setFile: (file: File) => void;
+    fileName: string | undefined;
+}
+
+export const BrowseFiles: React.FunctionComponent<BrowseFilesProps> = ({ setFile, fileName }) => {
+    const { getInputProps, getRootProps } = useDropzone({
+        onDrop: (files: File[], _rejected, _event) => {
+            console.log(files);
+            setFile(files[0]);
+        },
+    });
+    return (
+        <div style={{ padding: 20 }}>
+            {!fileName ? (
+                <div
+                    {...getRootProps()}
+                    style={{
+                        borderStyle: "solid",
```

Comment:
should move all inline css into scss modules files

https://stackoverflow.com/a/67353154

Reply:
I think ill leave this as is - because its just a sample and this was easier for me